Holovibes/sources/compute/converts.cc

```diff
@@ -393,4 +393,31 @@ void Converts::insert_complex_conversion(BatchInputQueue& input_queue)
 
     fn_compute_vect_->push_back(conversion_task);
 }
+
+void Converts::insert_float_dequeue(BatchInputQueue& input_queue, void* output)
+{
+    LOG_FUNC(fd_.depth);
```

Comment:
it is usefull to log the fd_.depth ?

Reply:
No, but it was a copy/paste from another function
I removed the `fd_.depth` from the `LOG_FUNC`.